README.md

```diff
@@ -34,10 +34,16 @@ These are the steps to take to install pyBDY:
   conda deactivate
   ```
 
-- Make sure the Java Runtime Environment is set (e.g. livljobs\*):
+- Make sure the Java Runtime Environment is set E.g.:
 
   ```
-  export JAVA_HOME=/usr/lib/jvm/java-1.8.0-openjdk-1.8.0.322.b06-1.el7_9.x86_64/
+  export JAVA_HOME=/usr/lib/jvm/java-1.8.0-openjdk-1.8.0.372.b07-1.el7_9.x86_64/ # e.g. for livljobs\*
+  ```
+
```

Comment:
You could also add the following to help the user (of *NIX architectures):
`readlink -f $(which java)`

Reply:
what about replacing the static path with this command?

Reply:
Made some changes. See what you think.